umm-spec-lib/src/cmr/umm_spec/validation/data_date.clj

```diff
@@ -0,0 +1,44 @@
+(ns cmr.umm-spec.validation.data-date
+  "Defines validations for UMM collection DataDates and MetadataDates"
+  (:require
+   [clj-time.core :as time]
+   [clojure.string :as string]
+   [cmr.umm-spec.date-util :as date-util]))
+
+(defn data-dates-warning-validation
```

Comment:
I am saying you could put additional tests in the umm-spec validation unit tests and cover all the scenarios. Easier to put them in there.